src/main/java/Yukee.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Scanner;
+
+public class Yukee {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
```

Comment:
Although 100 is a big enough size for the Task array, for scalability, you could explore using ArrayList which is dynamic